.github/workflows/ubuntu.yml

```
@@ -56,6 +53,15 @@ jobs:
       with:
         name: html
         path: html.tar.gz
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.4'
+        bundler-cache: true
+        
+      - name: Install Ruby dependencies
+        run: bundle install
```

Probably :p - sorry for the mess

```suggestion
    - name: Install Ruby dependencies
      run: bundle install
```